lib/datapacksutils.js

```diff
@@ -4861,6 +4862,21 @@ DataPacksUtils.prototype.formatSalesforceDeployResponse = function (response) {
     return currentStatusObj;
 }
 
+DataPacksUtils.prototype.formatSalesforceDeployResponseFromCLI = function (response) {
+    VlocityUtils.report("Source deploy response is ",response);
+    let currentStatusObj = {};
+    
+    if (response.details.componentSuccesses) {
+        response = response.details.componentSuccesses;
+    }
+     for( component of response){
```

Comment:
Will response be always an array? What if any exception or error occurs?
Above if can be simply `response = response?.details?.componentSuccesses;`

Reply:
This change has been accommodated